### ISSUE TITLE
SPA Static Route serves "Hello World" only

### ISSUE THREAD

**Issue by jtbaker** [taoufik07/responder#410]
```
import responder

api = responder.API(
    templates_dir="templates",
    static_route="/static",
    static_dir="static"
)

api.add_route("/", static=True)
```
On `v2.0+`, the index route at `/` only returns text of "Hello World", even though I have a valid `index.html` in the `/static/` directory. Downgrading to `1.3` fixed my issue for the time being. 

I might be able to dig into this and submit a PR this week if I can find a little time.

**Comment by daphil19** [comment]
#406 has the fix for this, but is awaiting direction from @taoufik07. Currently, the workaround in 2.0.3 is to use `api = responder.API(static_dir="static")` _without_ using `api.add_route("/", static=True)` and then navigating to `localhost:5042/static/index.html`.

**Comment by jtbaker** [comment]
Thanks for the response @daphil19! I think I'll just stick with the working `1.3` until #406 makes it into the release, so I won't have to make any codebase changes.

**Comment by taoufik07** [comment]
Thank you guys ! #406 has been released under v2.0.4 !
